profiles/common/modules/custom/multisite_drupal_features_set_standard/multisite_drupal_features_set_standard.info

```diff
@@ -50,6 +50,14 @@ feature_sets[nexteuropa_integration][font] = 'multisite-icon-share'
 feature_sets[nexteuropa_integration][enable][] = nexteuropa_integration
 feature_sets[nexteuropa_integration][disable][] = nexteuropa_integration
 
+feature_sets[nexteuropa_varnish][title] = 'Rule-based web frontend cache purging'
+feature_sets[nexteuropa_varnish][description] = 'Define rules to trigger granular purging of the web frontend caches'
+feature_sets[nexteuropa_varnish][documentation] = ''
```

Comment:
I've added a link to the internal wiki page as it is more complete and user oriented.